class-gwiz-gf-openai.php

```diff
@@ -1238,6 +1238,8 @@ public function maybe_save_result_to_field( $feed, $entry, $form, $text ) {
 
 		GFAPI::update_entry_field( $entry['id'], $map_result_to_field, $text );
 
+		gf_do_action( array( 'gf_openai_post_save_result', $form['id'] ), $text );
```

Comment:
Thoughts on the following, just so it's more verbose? We save the result in other places such as notes, etc.

```suggestion
		gf_do_action( array( 'gf_openai_post_save_result_to_field', $form['id'] ), $text );
```